django/river/common/database_connection/db_connection.py

```diff
@@ -47,8 +47,12 @@ def build_db_url(db_config):
                 "db_config specifies the wrong database model. "
                 "Only 'POSTGRES', 'ORACLE' and 'MSSQL' are currently supported."
             )
-
-        return f"{db_handler}://{login}:{password}@{host}:{port}/{database}{url_suffix}"
+        try:
+            [connection_type, target_name] = database.split(":", 1)
+            if connection_type == "service":
```

Comment:
can you add a `else` case raising a `NotImplementedError` to avoid returning None is someone use something different than "service" as connection_type ?